packages/trader/src/Modules/Trading/Components/Elements/mobile-widget.jsx

```diff
@@ -1,28 +1,17 @@
 import React from 'react';
 import { Money } from '@deriv/components';
-import { localize, Localize } from '@deriv/translations';
+import { localize } from '@deriv/translations';
```

Comment:
i think we are trying to keep `<Localize />` and deprecate `localize`

Reply:
Correct. But I removed code, which was using `<Localize />` from this file. 
`localize` was replaced in the 6-th package (https://github.com/binary-com/deriv-app/pull/10018/files#diff-2b788280f68fde149693030410289e8bc3773f737f6e8548f2fb2865428e0973) and I'm waiting for the end of sanity testing.

Reply:
I mean in that PR I replaced localize ---> <Localize />